src/thunderbots/software/test/ai/evaluation/calc_best_shot.cpp

```diff
@@ -0,0 +1,166 @@
+/**
+ * This file contains the unit tests for evaluation functions
+ * in calc_best_shot.cpp
+ */
+
+#include "ai/hl/stp/evaluation/calc_best_shot.h"
+
+#include <gtest/gtest.h>
+
+#include "geom/util.h"
+#include "shared/constants.h"
+
+TEST(CalcBestShotTest, calc_best_shot_with_vector_of_points_test)
+{
+    Field f                      = Field(10, 8, 10, 8, 3, 1, 1);
```

Comment:
Just a general note, we have a utility function to construct a "standard" Division B field.

`::Test::TestUtil::createStandardDivisionBField()` (or something close to that). It's just an easier and "standard" of way of making fields for tests. The field you have here isn't technically the "real" size we use. This doesn't really matter for tests, but just figured I'd point this out.

(If you do change this, this applies to all the tests)

Reply:
Changed to use `::Test::TestUtil::createStandardDivisionBField()` to create fields for all tests applicable